net/openvswitch/files/openvswitch.init

```diff
@@ -65,3 +67,22 @@ ovs_xx() {
 			;;
 	esac
 }
+
+ovs_bridge_init() {
+	local cfg="$1"
+
+	local disabled
+	local bridge_name
+	local of_controller
+	local of_controller_port
+
+	config_get_bool disabled "$cfg" disabled 0
+	[ "$disabled" -le 0 ] || return
```

Comment:
`config_get_bool` will set it to either `1` or `0`, that's part of the API and convention.  `[ "$disabled" = "0" ]` should do

Reply:
Agreed. Should I make this change for all the options "disabled" in package openvswitch ?

Reply:
Do as you will ;)

Reply:
> `config_get_bool` will set it to either `1` or `0`, that's part of the API and convention. `[ "$disabled" = "0" ]` should do

It may be desirable to apply this globally.
The "-le" operator is arithmetic, so the string argument presumable is parsed as an int.
The "==" operator is a string comparison, so should be computationally cheaper for these single char options.

( I won't make any global changes in this pull request... )